zulipterminal/model.py

```diff
@@ -603,6 +603,17 @@ def fetch_message_history(self, message_id: int,
         display_error_if_present(response, self.controller)
         return list()
 
+    def fetch_raw_message(self, message_id: int) -> str:
+        """
+        Fetches raw message content of a message using its ID.
+        """
+        response = self.client.get_raw_message(message_id)
+        if response['result'] == 'success':
+            return response['raw_content']
+        display_error_if_present(response, self.controller)
+
+        return "Raw content unavailable."
```

Comment:
I believe it would be reasonable to return an empty string so that you can check elsewhere for a failure easily.